app/mailers/reminder_mailer.rb

```diff
@@ -0,0 +1,27 @@
+class ReminderMailer < MandrillMailer::TemplateMailer
+  default from: "reminder@verba-app.com"
+
+  def send_reminder(user)
+    mandrill_mail(build_mail(to: [user],
+                             reminder_copy: ReminderCopy.copy_for(user)))
+  end
+
+  private
+  def build_mail(fields)
+    reminder_copy = fields.delete(:reminder_copy)
+
+    # Required fields
+    fields[:to].map!{ |user| {email: user.email, name: user.username} }
+    fields[:template] = 'daily-reminder'
```

Comment:
Yep, right, and It will provide the merge-vars:
CONTENT: which will contain the reminder email copy, 
ACTION: contains the 'call for action' field.
WEEK: contains the week number
DAY: the week day number.

This fields can be used in the template.